Makefile

```diff
@@ -94,9 +92,8 @@ demo: preinstall
 	@echo "Fidesops Privacy Center running at http://localhost:4000"
 	@echo "Fidesops Admin UI running at http://localhost:3000/login (user: fidesopsuser, pass: fidesops1A!)"
 	@echo "Example eCommerce demo app running at http://localhost:2000 (user: user@example.com, pass: user)"
+	@open demo.html
```

Comment:
I'm not sure if this'll work on Windows... hopefully it's the last step?

Reply:
`python3` is not a thing on windows generally, as unlike OSX there is no system-level python already installed. I had to change the command to just `python`

`./venv/bin/pip` did not work

`FLASK_APP=flaskr` etc. does not work

as suspected, `open` isn't a thing

definitely some good progress here but this is still a very heavily UNIX-reliant setup. Nox could help with most of this but I know that's a much more significant lift.